outlook/indico_outlook/plugin.py

```diff
@@ -122,7 +138,9 @@ class OutlookPlugin(IndicoPlugin):
     default_user_settings = {
         'enabled': True,  # XXX: if the default value ever changes, adapt `get_participating_users`!
         'status': None,
-        'status_overrides': [],
```

Comment:
Renaming the setting will require an alembic revision that updates it in the database (hopefully it's backwards compatible, otherwise it needs either some Postgres JSONB magic to convert the data - or just query, do it in python and update one by one - perfectly fine here as well)

You can run `indico db --plugin outlook revision -m 'Update user setting'` to create an empty revision file, and to execute it run `indico db --plugin outlook upgrade`

Reply:
Done!